### DISCUSSION TITLE
Fundamentals Part 2: Way too much info for one lesson

### DISCUSSION THREAD

**Discussion by dandruffking** [TheOdinProject/curriculum#26888] [General]
### Describe your suggestion

I love "TOP" so far, but there is way too much important information in this lesson for anyone new to programming to feasibly absorb it. It's overwhelming and imo should be broken up into AT LEAST two sections to allow for better comprehension and retention. 

There are **_9_** assigned articles that are very in depth and important to JS, after getting through article 7 (it's taken me almost 3 days to get to this point in this lesson) I've already lost some of the information in the earlier articles.  

Please consider this suggestion and break it up into more manageable pieces.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-fundamentals-part-2

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

**Comment by 01zulfi** [comment]
@dandruffking Thanks for the suggestion.

@TheOdinProject/foundations, what do y'all think?
